### PR TITLE
review padding at edges of screens

### PR DESCRIPTION
**Pull request by axemonkey**
Edge padding was inconsistent across different areas of the site. GDS guidelines suggest 30px on large screens, and 15px on smaller, though with the amount of data we have to present on some pages 30px is a bit much, so I've gone with 15px across the board.

### CODE REVIEW

File: dashboard/assets/common/common.css [hunk 1]
```diff
@@ -1,18 +1,3 @@
-#content {
```
Comment (cliffxuan) [inline dashboard/assets/common/common.css:1]: why is this moved to `main.css`? i'm guessing this is the better place for it. `main.css` is only the style for js front end, which is a sub section of the `#content` section.

Comment (axemonkey) [reply dashboard/assets/common/common.css:1]: OK so `main.css` affects the non Django-admin front-end, and `common.css` is, as the name suggests, common to both applications. In the main dashboard app we don't have any pages (to my knowledge) where there are tables that stretch from edge to edge when the screen is small (e.g. 900px) but in admin, we do. So I moved the chunk of CSS that was applying padding globally and reapplied it to sections of admin that need it.

I see now that I also moved the min-height and width bits without realising. I shall move those back. 😃

Comment (cliffxuan) [reply dashboard/assets/common/common.css:1]: I see. Makes sense.

**Comment by cliffxuan** [comment]
looks good
